### PR TITLE
Storage documentation

### PR DESCRIPTION
**Pull request by g4s8**
For #860 - described FS and S3 storages, added todo for others, described default storages

### CODE REVIEW

File: .wiki/Configuration-Storage.md [hunk 1]
```diff
@@ -1 +1,70 @@
-TBD
+# Storage
+
+Artipie "Storage" is an abstraction on top of multiple key-value storage providers. Artipie supports:
+ - file system storage
+ - S3 storage
+ - etcd storage (see limitations)
```
Comment (olenagerasimova) [inline .wiki/Configuration-Storage.md:6]: @g4s8 seems like word `limitations` requires some link

**Review by olenagerasimova — CHANGES_REQUESTED** [review]
@g4s8 thanks, just one comment

**Comment by g4s8** [comment]
@olenagerasimova thanks, described missed storages and limitations

**Comment by olenagerasimova** [comment]
@g4s8 please, could you check that push passed? Do not see any changes since previous push

**Comment by g4s8** [comment]
> @g4s8 please, could you check that push passed? Do not see any changes since previous push

@olenagerasimova sorry, didn't push, please see latest https://github.com/artipie/artipie/pull/916/commits/fdae7d8e56abcf405dd266bb69058f3b8a9b586e

**Review by olenagerasimova — APPROVED** [review]
@g4s8 thanks, approving

**Pull request merged**